workflow_imaging/pipeline.py

```diff
@@ -34,6 +34,12 @@ class Session(dj.Manual):
     session_datetime: datetime(3)
     """
 
+    class Directory(dj.Part):
+        definition = """
+        -> master
+        session_dir: varchar(256)       # Absolute path
```

Comment:
this should be relative path, relative to the `imaging_root_data_dir`

Reply:
i agree that we should store the relative path in the table, but am a bit confused as to why we should store the absolute path in the csv file.  thanks

Reply:
I think that offers better usability, usually users would just copy and paste the whole path there, it's a bit much to have to expect from them to relativize the path first before placing into the csv file

Reply:
makes sense, thanks